app/views/reports/load_task_data_in_report.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#tasks_table').html('')
+$('#tasks_table').append("<%= j render 'shared/tasks_report_list' %>")
```

Comment:
try to add a newline at the end of all files as a standard practice (as rubocop sugests)